README.md

```diff
@@ -22,6 +22,12 @@ You can then run `helm search repo jenkins` to see the charts or obtain an exhau
 <!-- Keep full URL links to repo files because this README syncs from main to gh-pages.  -->
 Chart documentation is available in [jenkins directory](https://github.com/jenkinsci/helm-charts/blob/main/charts/jenkins/README.md).
 
+## Building weekly releases
+
+The default charts target Long-Term-Support (LTS) releases of Jenkins.
+
+You can build charts for weekly releases yourself by replacing `appVersion` in [`charts/jenkins/Chart.yaml`](https://github.com/jenkinsci/helm-charts/blob/14e78d66107c61341fbb843f156f6e4e30238c65/charts/jenkins/Chart.yaml#L5) with the desired version and running `helm package charts/jenkins`.
```

Comment:
I thinks it's worth noting that version should also be adjusted in that case as one should try to avoid using same version numbers as already released charts.

Reply:
Good point!